jobs/payment-jobs/utils/mailer.py

```diff
@@ -106,7 +106,7 @@ def publish_statement_notification(pay_account: PaymentAccountModel, statement:
 def publish_payment_notification(info: StatementNotificationInfo) -> bool:
     """Publish payment notification message to the mailer queue."""
     message_type = QueueMessageTypes.PAYMENT_DUE_NOTIFICATION.value \
-        if info.action == StatementNotificationAction.OVERDUE \
+        if info.action == StatementNotificationAction.DUE \
```

Comment:
in Due / Overdue?

Reply:
Good call, I will add an IN to be inclusive of both